helios-tools/src/test/java/com/spotify/helios/cli/command/RollingUpdateCommandTest.java

```diff
@@ -524,6 +491,94 @@ public void testRollingUpdateOverlapJson() throws Exception {
     );
   }
 
+  @Test
+  public void testCommandLineOptions() throws Exception {
+    when(client.rollingUpdate(anyString(), any(JobId.class), any(RolloutOptions.class)))
+        .thenReturn(immediateFuture(new RollingUpdateResponse(RollingUpdateResponse.Status.OK)));
+
+    when(client.deploymentGroupStatus(GROUP_NAME)).then(new ResponseAnswer(
+        statusResponse(DeploymentGroupStatusResponse.Status.ACTIVE, null,
+            makeHostStatus("host1", JOB_ID, TaskStatus.State.RUNNING))
+    ));
+
+    String optionString = "(parallelism=1, timeout=300, overlap=false, token=my_token, "
+                          + "ignoreFailures=false)";
+
+    command.runWithJob(options, client, out, false, JOB, null);
+
+    assertThat(baos.toString(), containsString(optionString));
+  }
+
+  @Test
+  public void testFallbackToJobOptions() throws Exception {
```

Comment:
👍  You can also set some of the CLI args to check it does a merge of both CLI args and job rollout options with correct precedence.